docs/reference/commandline/service_rollback.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "service rollback"
+description: "The service rollback command description and usage"
+keywords: "service, rollback"
+---
+
+<!-- This file is maintained within the docker/cli Github
+     repository at https://github.com/docker/cli/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# service rollback
+
+```Markdown
+Usage:	docker service rollback SERVICE
+
+Rollback a service
+
+Options:
+  -d, --detach       Exit immediately instead of waiting for the service to converge (default true)
+      --help         Print usage
+  -q, --quiet        Suppress progress output
+```
+
+## Description
+
+Rollback a specified service from the swarm. This command has to be run
+targeting a manager node.
+
+## Examples
+
+Rollback the `redis` service:
+
+```bash
+$ docker service create --name redis redis:3.0.6
```

Comment:
I wonder if "updating" the image from `3.0.6` to `3.0.7`, then rolling back could make people think the "rollback" is rolling back an image update, not the service definition. Although it makes a nice example to update the image version, and roll back that change, would it be better to use something else for illustration? e.g.

    ### Roll back a service to its previous version

    This example creates a service with a single replica, updates the
    service to use three replicas, and then rolls back the service to the
    previous version, having one replica.

    Create a service with a single replica:

    ```bash
    $ docker service create --name my-service -p 8080:80 nginx:alpine
    ```

    Confirm that the service is running with a single replica:

    ```bash
    $ docker service ls

    ID                  NAME                MODE                REPLICAS            IMAGE               PORTS
    xbw728mf6q0d        my-service          replicated          1/1                 nginx:alpine        *:8080->80/tcp
    ```

    Update the service to use three replicas:

    ```bash
    $ docker service update --replicas=3 my-service

    $ docker service ls

    ID                  NAME                MODE                REPLICAS            IMAGE               PORTS
    xbw728mf6q0d        my-service          replicated          3/3                 nginx:alpine        *:8080->80/tcp
    ```

    Now rollback the service to its previous version, and confirm it is
    running a single replica again:

    ```bash
    $ docker service rollback my-service

    $ docker service ls

    ID                  NAME                MODE                REPLICAS            IMAGE               PORTS
    xbw728mf6q0d        my-service          replicated          1/1                 nginx:alpine        *:8080->80/tcp
    ```